components/src/Branding/Branding.js

```diff
@@ -74,15 +75,33 @@ const BrandingWrap = styled.div(
     }
   }),
   ({ size }) => ({
-    fontSize: size === "large" ? theme.fontSizes.large : theme.fontSizes.small,
-    lineHeight: size === "large" ? theme.lineHeights.subsectionTitle : theme.lineHeights.smallTextCompressed
+    padding: size === "large" ? null : "4px 0"
   })
 );
 
-const BaseBranding = ({ logoType, subtext, size, alignment, logoColor, ...props }) => (
+const Line = styled.div(({ logoColor }) => ({
+  position: "relative",
+  width: "100%",
+  ":before": {
+    content: "''",
+    position: "absolute",
+    top: "50%",
+    left: 0,
+    borderTop: `1px solid ${getLogoColor(logoColor).line}`,
+    background: getLogoColor(logoColor).line,
+    width: "100%",
+    transform: "translateY(-50%)"
+  }
+}));
+
+const BaseBranding = ({ logoType, subtext, size, alignment, withLine, logoColor, ...props }) => (
   <BrandingWrap size={size} logoColor={logoColor} alignment={alignment} {...props}>
     {logoType === "lettermark" && (
-      <svg {...getSize(size, logoType)} viewBox="0 0 37 32" style={{ display: "block" }}>
+      <svg
```

Comment:
Could you extract the two logos into their own components?
They don't need to be public.

I think it will help a lot with readability of the main render function.

Also, it will simplify the `getSize` logic. :)